### DISCUSSION TITLE
Does the 'border' attribute work?

### DISCUSSION THREAD

**Discussion by YouCantCheatOnVACSecuredServers** [mermaid-js/mermaid#6011] [Q&A]
I want to add a database (a rectangle without left and right borders like in DFD - Data Flow Diagram notation) to a flowchart diagram using a 'border' attribute. A 'border' attribute was added as a result of this issue: https://github.com/mermaid-js/mermaid/issues/1893

However the following code draws a full rectangle and not a rectangle without left and right borders:

```
flowchart LR
    db[|borders:tb| database]
```

```mermaid
flowchart LR
    db[|borders:tb| database]
```

Does border attribute work? Did it stop working at some point of time?

**Comment by nprime496** [comment]
I have the same question.
